recipes/opencv/4.x/conanfile.py

```diff
@@ -290,7 +307,18 @@ def _configure_cmake(self):
         self._cmake.definitions["WITH_ARAVIS"] = False
         self._cmake.definitions["WITH_CLP"] = False
         self._cmake.definitions["WITH_NVCUVID"] = False
-        self._cmake.definitions["WITH_FFMPEG"] = False
+
+        self._cmake.definitions["WITH_FFMPEG"] =\
+                "ON" if self.options.get_safe("with_ffmpeg") else "OFF"
+
+        if self.options.get_safe("with_ffmpeg"):
+            self._cmake.definitions["OPENCV_FFMPEG_SKIP_BUILD_CHECK"] = "ON"
+            self._cmake.definitions["OPENCV_FFMPEG_SKIP_DOWNLOAD"] = "ON"
+            # opencv will not search for ffmpeg package, but for
+            # libavcodec;libavformat;libavutil;libswscale modules
+            self._cmake.definitions["OPENCV_FFMPEG_USE_FIND_PACKAGE"] = "OFF"
+            self._cmake.definitions["OPENCV_INSTALL_FFMPEG_DOWNLOAD_SCRIPT"] = "OFF"
```

Comment:
what about https://www.python.org/dev/peps/pep-0008/#maximum-line-length ?

Is it better not to follow that rule because the rest of the file does not respect it?

Reply:
True/False did not work, I had to use "ON" and "OFF" to make CI happy.

Reply:
Which error?

Reply:
let's try

Reply:
Well, it was another problem  then! your modifications are applied and everything seem OK.